contracts/connectors/loantoken/modules/LoanTokenSettingsLowerAdmin.sol

```diff
@@ -193,9 +195,14 @@ contract LoanTokenSettingsLowerAdmin is LoanTokenLogicStorage {
 					uint256(0xd46a704bc285dbd6ff5ad3863506260b1df02812f4f857c8cc852317a6ac64f2)
 				)
 			);
+		assembly {
+			paused := sload(slot)
+		}
+		require(paused != isPaused, "invalid");
 		assembly {
 			sstore(slot, isPaused)
 		}
+		emit ToggleFunctionPaused(funcId, !isPaused, isPaused);
```

Comment:
the event should rather be called
ToggledFunctionPause
(toggle is the verb here)

Reply:
Done